cloud/policies/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Policies in Terramate Cloud
+description: Learn how Terramate Cloud uses policies to manage your infrastructure reliably
+---
+
+# Terramate Cloud Policies
+
+Terramate Cloud policies help secure your infrastructure by detecting misconfigurations on a per-resource level. They leverage the industry-standard CIS Benchmarks to automatically analyze your infrastructure as code (IaC) changes and deployed resources.
+
+## What Do Policies Do?
+
+- **Misconfiguration Detection:** Policies inspect each resource for potential security misconfigurations. This precise, per-resource analysis helps you catch vulnerabilities early.
+- **Automated Analysis:** When you deploy changes or run scheduled [drift detection](../drift/synchronization-in-automation.md), policies automatically verify that your configurations meet the prescribed security standards.
```

Comment:
e.g. here we should rather link to drift detection as a concept in TMC https://terramate.io/docs/cloud/drift/